src/app/ui/nav/NavItem.module.scss

```diff
@@ -4,8 +4,14 @@
 
```

Comment:
I'd replace the top/bottom margins with padding. When mousing over the list of NavItems, there are spaces between them that cause the arrow to jump back to the active item. Removing the margins will prevent it from jumping